lib/Dialect/Torch/Transforms/DecomposeComplexOpsEarly.cpp

```diff
@@ -33,13 +33,13 @@ class DecomposeAtenSplitSizesOp : public OpRewritePattern<AtenSplitSizesOp> {
     SmallVector<int64_t> splitSizes;
     // TODO: Support non static splitSizes.
     if (!matchPattern(op.split_size(), m_TorchConstantIntList(splitSizes)))
-      return failure();
+      return rewriter.notifyMatchFailure(op, "int list does not exist");
```

Comment:
The error message is not accurate. The right thing to say is something along the lines of `op.split_sizes() must be a constant int list`. You can do `git grep m_TorchConstantIntList` to see example error messages other people have used.